desk/sur/groups.hoon

```diff
@@ -3,7 +3,7 @@
 /-  grp=group-store
 /-  metadata-store
 |%
-++  okay  `epic:e`1
```

Comment:
Not clear to me what's up with this and the v3 marks. We didn't change the types or subscription API, right?

Reply:
yeah this was the reason, no state changes just a semi-convenient way to break the subscription